NLP/stopwords.csv

```diff
@@ -1,2 +1,7 @@
 0,kan
-1,kom
\ No newline at end of file
+1,kom
```

Comment:
disse ord slipper af en eller anden grund igennem systemet, så jeg har tilføjet dem som stopord, for stoppe dem i at være en ting.
Ser ud til at der er et mindre problem med et-bogstavs ord, men syntes ikke det er værd at lave en større fix til